molecule_common/origin-latest/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openshift/origin:latest
```

Comment:
Perhaps we should be targeting an specific release version instead of `:latest`.